### DISCUSSION TITLE
A B X Y buttons wrong mapping Linux mint

### DISCUSSION THREAD

**Discussion by oldmanagain** [AntiMicroX/antimicrox#883] [Q&A] [answered]
The buttons on the Linux app have wrong assignments —- they appear to be reversed.  I can’t find a way to correct them.  Anybody know?

**Comment by pktiuk** [comment, marked as answer]
Use `Controller Mapping` button

**Reply by oldmanagain** [reply]
I guess I didn't phrase it correctly....

....when I press A antimicrox show B and when I press X it shows Y.

**Reply by oldmanagain** [reply]
My bad --- this is why 'old men' shouldn't be doing this stuff....

....THANKS for setting me right!
